.npmignore

```diff
@@ -0,0 +1,10 @@
+target
+Cargo.lock
+.cargo
+.github
+npm
+.eslintrc
+.prettierignore
+rustfmt.toml
+yarn.lock
```

Comment:
These aren't used/produced by anything, are they? (Except `.rustfmt.toml` but it's missing the leading dot.)

Reply:
But mostly this is here because it was generated by `napi-rs`, so if it's really not needed then it could go